Sources/Sentry/Public/SentryOptions.h

```diff
@@ -200,6 +200,8 @@ NS_SWIFT_NAME(Options)
 @property (nonatomic, assign) BOOL enableUIViewControllerTracking;
 
 /**
+ * This feature is EXPERIMENTAL.
```

Comment:
Personally I would add this sentence underneath the "Automatically attaches a screenshot when capturing an error or exception." sentence. So first you describe what it does, then you mean mention it's experimental. 

If you agree, best to also change it for attachViewHierarchy

Reply:
Too late 😅

Reply:
I was just following what we have in other comments.
I dont have a strong opinion on this.

Reply:
Fair enough! Let's leave it like this.